lib/data/model.ts

```diff
@@ -169,6 +169,10 @@ export default class Model extends DenaliObject {
     return new Proxy(this, {
 
       get(model: Model, property: string): any {
+        if (property === 'constructor') {
```

Comment:
and since any POJO is an instance of Object (meaning it has a constructor), it doesn't even have to be an instance of a class if you just straight return a property from the underlying model record.